content/en/docs/concepts/overview/components.md

```diff
@@ -23,14 +23,45 @@ Here's the diagram of a Kubernetes cluster with all the components tied together
 {{% /capture %}}
 
 {{% capture body %}}
-## Control Plane Components
+## Node components
 
-The Control Plane's components make global decisions about the cluster (for example, scheduling), as well as detecting and responding to cluster events (for example, starting up a new {{< glossary_tooltip text="pod" term_id="pod">}} when a deployment's `replicas` field is unsatisfied).
+In Kubernetes, your application runs inside a group of containers called
+a {{< glossary_tooltip text="_pod_" term_id="pod" >}}. The idea of a pod
+is to isolate different workloads from each other. The computers
```

Comment:
Technically, containers isolate and pods enable sharing of resources between containers within a pod. I would just eliminate the "idea of a pod" sentence, and change the previous sentence to "group of colocated containers".

Reply:
Can you clarify the change you're suggesting @bgrant0607? The aim of this PR is to introduce the Pod concept, so this is the crucial bit.

Issue https://github.com/kubernetes/website/issues/14724 is also relevant.

Reply:
Thanks for the pointer to that issue.

It depends on the perspective one is coming from, and what the core "idea" of a pod is, vs. attributes of pods.

The original Pod documentation:
https://github.com/kubernetes/kubernetes/blob/release-0.4/docs/pods.md

made it into:
https://kubernetes.io/docs/concepts/workloads/pods/pod/

And was re-explained here, in a way that emphasized its use by the workload APIs:
https://kubernetes.io/docs/concepts/workloads/pods/pod-overview/

At the time Kubernetes was created, most container-oriented systems used containers as the minimal units of provisioning, scheduling, execution, addressing, isolation, and so on. In Kubernetes, it is the Pod that is the minimal unit. Containers within a pod may share resources. That sharing (e.g., a single IP address) was a big difference compared to other systems.

So, isolation is one of many aspects that Kubernetes moves from the container level to the pod level. It's going to be hard to be both concise and clear in this brief intro. I suggest linking https://kubernetes.io/docs/concepts/security/overview/ directly to the pod documentation rather than or in addition to this document.

That said, I propose something like the following:
In Kubernetes, your application runs inside groups of one or more co-located containers called pods. Containers within a pod may share resources, such as an IP address and storage volumes, whereas pods are isolated from one another and from their hosts. The hosts that run pods are called nodes, and one node can run pods from many different applications.